integrations/issue_stress_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"testing"
+
+	"code.gitea.io/gitea/models"
+)
+
+func TestIssueNoDupIndex(t *testing.T) {
+	prepareTestEnv(t)
+
+	models.TestIssueNoDupIndex(t)
```

Comment:
Why recall unit test here ?
Either restore integrations test (with req handling) or just drop this file.

Reply:
I wanted the unit test to be called from all types of database in the drone. It was the easiest way. The unit tests normally run only for SQLite.